src/2_RTB/PianoDiQualifica/content.typ

```diff
@@ -164,14 +165,473 @@ Sono state individuate due metriche principali:
+      ..test_di_sistema.map(item => (item.at(0),item.at(3))).flatten().map(item => [#item])
+),caption: "Tracciamento dei test di sistema.")
 
 
+== Liste di controllo
 
+Le liste di controllo sono uno strumento che i verificatori possono utilizzare al fine di individuare errori ricorrenti nella documentazione. Tali liste prevedono anche una descrizione del problema al fine di poter fornire una spiegazione della richiesta di cambiamenti durante la fase di revisione.
+Le liste di controllo possono essere aggiornate durante tutto il corso del progetto dai Verificatori, man mano che vengono notati errori ricorrenti.
 
+#pagebreak()
+=== Struttura dei documenti
+
+#figure(
+table(
+      columns: (1fr, 2fr),
+      inset: 10pt,
+      align: (center, left),
+      [*Aspetto*],[*Spiegazione*],
+      [Vuoti documentativi],[Tutte le sezioni che non si intende completare, e che quindi devono rimanere vuote, devono essere cancellate.],
+      [Didascalia assente],[Tutte le tabelle e le immagini devono avere una didascalia descrittiva.],
+      [Ripetizione di elementi ricorrenti],[Ogni elemento che viene usato più di una volta, soprattutto se si sa a priori che verrà usato più volte, deve essere creato da una funzione #glossary("Typst").],
+      [Ordine non alfabetico],[I nomi dei componenti devono essere riportati in ordine alfabetico, in qualsiasi documento dove compaiono come elenco non numerato.],
+      [Aggiornamento fantasma],[Ad ogni insieme di modifiche ai documenti che devono essere aggiunte assieme, deve corrispondere una riga nella tabella del changelog, con un univoco numero di versione.],
+      [Titolo principali in mezzo alla pagina],[Tutti i titoli principali devono iniziare la pagina nella quale vengono inseriti.],
+
+),
+caption: [Lista di controllo per la struttura dei documenti]
+)
 
+#pagebreak()
+=== Errori ortografici, di lingua italiana e di forma
+
+#figure(
+table(
+      columns: (1fr, 2fr),
+      inset: 10pt,
+      align: (center, left),
+      [*Aspetto*],[*Spiegazione*],
+      [Errori di battitura],[Gli errori di battitura, di distrazione o di ignoranza devono essere rimossi.],
+      [Errori di coniugazione],[Gli errori di coniugazione devono essere rimossi.],
+      [Forma verbale],[Il presente indicativo è da preferire.],
+      [Forma non concise],[Le espressioni troppo verbose, ove possibile, devono essere ridotte a forme più concise.],
+      [Non formalità],[Le espressioni non formali devono essere sostituite con le corrispondenti espressioni formali.],
+      [Richiamo errato al documento],[Ogni richiamo ai documenti deve seguire la seguente forma: _NomeDocumento\_vVersioneMajor.VersioneMinor_ (e.g. _PianoDiProgetto\_v1.0_).],
+      [Termini impropriamente in maiuscolo],[I termini di glossario non godono della proprietà di avere la prima lettera maiuscola, rispetto alle parole tradizionali. Seguono le stesse regole delle parole non di glossario in quanto alla forma (ovviamente non riguardo allo stile).],
```

Comment:
Neanche se nomi propri o all'inzio della frase?

Reply:
Non hai capito: "seguono le stesse regole delle parole non di glossario" vuol dire che se sono all'inizio della frase ovviamente le metti in maiuscolo. Ma se le posizioni in mezzo ad una frase lasci le lettere in minuscolo.